rust/noosphere-gateway/src/worker/syndication.rs

```diff
@@ -77,162 +77,175 @@ where
 
```

Comment:
No logic changes in this file, just copies the handling of `process_job()` like the NS worker